Dockerfile

```
@@ -38,7 +38,7 @@ RUN mkdir /srv/ttn-lorawan/public/blob
 VOLUME ["/srv/ttn-lorawan/public/blob"]
 
 ENV TTN_LW_BLOB_LOCAL_DIRECTORY=/srv/ttn-lorawan/public/blob \
-    TTN_LW_IS_DATABASE_URI=postgres://root@cockroach:26257/ttn_lorawan?sslmode=disable \
+    TTN_LW_IS_DATABASE_URI=postgresql://root@localhost:5432/ttn_lorawan_dev?sslmode=disable&user=root&password=root \
```

Doesn't this work?

```suggestion
    TTN_LW_IS_DATABASE_URI=postgresql://root:root@localhost:5432/ttn_lorawan_dev?sslmode=disable \
```

BTW: We're removing this in a separate pull request